source/DistanceAndDirection/DistanceAndDirectionLibrary/Views/LinesView.xaml.cs

```diff
@@ -21,9 +21,20 @@ namespace DistanceAndDirectionLibrary.Views
     /// </summary>
     public partial class GRLinesView : UserControl
     {
+        #region Properties
+
+        public ComboBox CmbLineType
```

Comment:
need to remove this, the proper way to do this is create a Visibility property in the view model then bind the elements visibility to the view model property.